docs/source/ecf.rst

```diff
@@ -49,6 +49,14 @@ linearity_file
 ''''''''''''''
 The fully qualified path to the custom linearity correction file to use if custom_linearity is True.
 
+custom_mask
+''''''''''''''''
+Boolean. If True, allows user to supply a custom bad pixel mask file and overwrite the default file.
+
+mask_file
+''''''''''''''
+The fully qualified path to the custom bad pixel mask file to use if custom_mask is True.
```

Comment:
Can you explain what format this file should have - should it be a CSV, FITS, etc. Should users provide indices for bad pixels, or is it an array of 0/1 of True/False and if so are the "truthy" values pixels that should be masked or pixels that should not be masked.

Reply:
@erinmmay, I need your response on the questions above to be able to adjust the docs myself

Reply:
custom mask file is just the pmask file from crds (fits file) with additional bad pixels marked by changing the pixel value to the "do not use" value